src/watcher.js

```diff
@@ -266,6 +267,40 @@ class Watcher extends SafeEventEmitter {
         this.on('load.chat', updateChannel);
         this.on('load.vod', updateChannel);
     }
+
+    tmiEventObserver() {
+        const PATCHED_SENTINEL = () => {};
+        let twitchConsumeEvent;
+        const watcher = this;
```

Comment:
would prefer a separate function for emitting an event than storing a reference to this.